config/locales/en.yml

```diff
@@ -886,6 +886,11 @@ en:
         service_plan:
           has_contracts: 'This service plan cannot be deleted! At least one contract depends on it.'
 
+        service:
+          attributes:
+            system_name:
+              invalid: invalid. Only ASCII letters, numbers, dashes and underscores are allowed.
```

Comment:
This is because the default message is just 'invalid'. But this way we are allowed to override any message we want for any specific error and not stick to whatever message for the entire attribute.